docusaurus/docs/learn/quickstarts/network/local-no-docker.md

```diff
@@ -8,19 +8,22 @@ quickly and easily, entirely locally. Since you'll be running everything locally
 between network components. All the processes will run locally, and you'll be responsible for starting and stopping them
 when you want to turn the overlay network on or off.
 
-## Preparation
+## Prerequisites
+
+:::note
+Make sure you have `jq` and `curl` installed before running the `expressInstall` one-liner. Both are available in `apt` (Debian) and `dnf` (RHEL, Rocky, Fedora) as eponymous packages.

```

Comment:
this 'note' seems like it could be reused. also, let's remove : "Both are available in `apt` (Debian) and `dnf` (RHEL, Rocky, Fedora) as eponymous packages."